components/dashboard/src/projects/Prebuild.tsx

```diff
@@ -52,7 +52,7 @@ export default function () {
         if (!prebuild) {
```

Comment:
nice catch!

Reply:
@jankeromnes, I've add a check to prevent it from hitting the db layer: https://github.com/gitpod-io/gitpod/pull/5116/files#diff-9c81815b0337b4e00a2ab1615cab6010a2497412f99dfc63c73a17f8bf28a90cR877

Can you invite me to that team please? I'd like to reproduce/verify the fix

Reply:
It's on my user 😬 but please feel free to move it to a team by hacking the DB